planning/autoware_behavior_velocity_crosswalk_module/plugins.xml

```diff
@@ -0,0 +1,3 @@
+<library path="autoware_behavior_velocity_crosswalk_module">
+  <class type="autoware_behavior_velocity_planner::CrosswalkModulePlugin" base_class_type="autoware_behavior_velocity_planner::PluginInterface"/>
+</library>
```

Comment:
:thinking: https://github.com/autowarefoundation/autoware.universe/blob/5a1ac0a3d7f63cb8ac7ccba20f7822cbcc13f56c/planning/autoware_behavior_velocity_crosswalk_module/src/manager.cpp#L235-L237

Reply:
Thank you
Sorry for requesting the review against an old version.
maybe I've fixed